### PR TITLE
Add Tokenomics block

### PR DESCRIPTION
**Pull request by kh-ruslan**

### CODE REVIEW

File: src/components/Tokenomics/Tokenomics.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+                            <span className="subtitle-2-18-700">
+                                1,000,000,000 YEL
+                            </span>
+                        </div>
+
+                        {pieChartItems.map(
+                            ({ percent, tokensCount, title, className }) => (
+                                <div
+                                    className={`tokenomics__pie-chart--item  ${className}`}
+                                >
```
Comment (plsthink) [inline src/components/Tokenomics/Tokenomics.js]:
```suggestion
                                <div
                                    key={title}
                                    className={`tokenomics__pie-chart--item  ${className}`}
                                >
```

**Review by plsthink — CHANGES_REQUESTED** [review]
Please:

- [x] Take care of responsiveness of the block: the image does not position correctly on lower body width.
- [x] Rename `Tokenomics.js` to `index.js` and `Tokenomics.scss` to `styles.scss`, in order to match other components naming convention.

**Comment by jengjeng** [comment]
@kh-ruslan pls check the breakpoint to be the same with http://yellow.org/

maybe you pull the latest from master first into your branch

File: src/components/Tokenomics/index.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+                            <p>
+                                Token will be pre-minted and distributed based
+                                on the information presented on the pie chart.
+                            </p>
+
+                            <div>
+                                <Link
+                                    className="button button--outline button--secondary button--shadow-secondary"
+                                    to="/docs/overview/what-is-xln"
```
Comment (josadcha) [inline src/components/Tokenomics/index.js]: please change link to `/docs/concepts/architecture`

File: src/components/Tokenomics/index.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+import Link from "@docusaurus/Link";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+export default function Tokenomics() {
+    return (
+        <div className="section section--grey">
+            <div className="container tokenomics">
+                <div className="row">
+                    <div className="col col--6">
+                        <span className="subtitle-with-bg-primary subtitle-2-18-700">
```
Comment (josadcha) [inline src/components/Tokenomics/index.js]: use` .highlight` class instead, remove styles for this class

**Review by josadcha — CHANGES_REQUESTED** [review]

**Review by josadcha — APPROVED** [review]

**Pull request merged**